contracts/src/v0.8/dev/vrf/VRFV2PlusWrapper.sol

```diff
@@ -384,11 +388,27 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
     lastRequestId = requestId;
   }
 
+  function checkPaymentMode(bytes memory extraArgs, bool isLinkMode) public pure {
+    if (extraArgs.length < 36) {
+      revert IncorrectExtraArgsLength(36, uint16(extraArgs.length));
+    }
+    bool nativePayment = extraArgs[35] == hex"01";
```

Comment:
a comment on this magic number would be helpful

Reply:
Added a comment